modules/rtm/R/invert.auto.R

```diff
@@ -218,17 +217,68 @@ combineChains <- function(samps1, samps2) {
     sampsfinal[[i]] <- rbind(samps1[[i]], samps2[[i]])
   }
   stopifnot(length(sampsfinal) == length(samps1))
```

Comment:
I dunno, I think between the call stack and the variable names, these are pretty self-explanatory. I'd have a hard time describing these errors in a way that this doesn't do already.